support-frontend/assets/helpers/subscriptionsForms/validation.js

```diff
@@ -33,6 +33,10 @@ function firstError<FormField>(field: FormField, errors: FormError<FormField>[])
 
 }
 
+function removeError<FormField>(field: FormField, formErrors: FormError<FormField>[]): FormError<FormField>[] {
```

Comment:
I love this - I much prefer forms that adapt to inputs post error (which is what I **believe** this is doing)

Reply:
And then I realised it was already in there - just somewhere else...!

Reply:
It is 😃